db/views/materialized_patient_summaries_v03.sql

```diff
@@ -0,0 +1,416 @@
+        a.scheduled_date as follow_up_date,
+        greatest(0, date_part('day', a.scheduled_date - date_trunc('day', a.device_created_at)))::int as follow_up_days,
+
+        pd_1.name as prescription_drug_1_name,
+        pd_1.dosage as prescription_drug_1_dosage,
+
+        pd_2.name as prescription_drug_2_name,
+        pd_2.dosage as prescription_drug_2_dosage,
+
+        pd_3.name as prescription_drug_3_name,
+        pd_3.dosage as prescription_drug_3_dosage,
+
+        pd_4.name as prescription_drug_4_name,
+        pd_4.dosage as prescription_drug_4_dosage,
+
+        pd_5.name as prescription_drug_5_name,
+        pd_5.dosage as prescription_drug_5_dosage,
+
+        other_medications.other_prescription_drugs,
+        other_medications.all_prescription_drugs,
+
+        rank() over (partition by bp.patient_id order by bp.recorded_at desc) rank
+    from blood_pressures bp
+    left outer join facilities f on bp.facility_id = f.id
+    left outer join appointments a on a.patient_id = bp.patient_id and date_trunc('day', a.device_created_at) = date_trunc('day', bp.recorded_at)
+    left outer join facilities follow_up_facility on follow_up_facility.id = a.facility_id
+    left outer join ranked_prescription_drugs pd_1 on bp.id = pd_1.bp_id and pd_1.rank = 1
+    left outer join ranked_prescription_drugs pd_2 on bp.id = pd_2.bp_id and pd_2.rank = 2
+    left outer join ranked_prescription_drugs pd_3 on bp.id = pd_3.bp_id and pd_3.rank = 3
+    left outer join ranked_prescription_drugs pd_4 on bp.id = pd_4.bp_id and pd_4.rank = 4
+    left outer join ranked_prescription_drugs pd_5 on bp.id = pd_5.bp_id and pd_5.rank = 5
+    left outer join other_medications on pd_1.bp_id = other_medications.bp_id
+    where bp.deleted_at is null and a.deleted_at is null
+), latest_blood_pressures as (
+    select
+        latest_blood_pressure_1.patient_id as patient_id,
+
+        latest_blood_pressure_1.id as latest_blood_pressure_1_id,
+        latest_blood_pressure_1.recorded_at as latest_blood_pressure_1_recorded_at,
+        latest_blood_pressure_1.systolic as latest_blood_pressure_1_systolic,
+        latest_blood_pressure_1.diastolic as latest_blood_pressure_1_diastolic,
+        latest_blood_pressure_1.facility_name as latest_blood_pressure_1_facility_name,
+        latest_blood_pressure_1.facility_type as latest_blood_pressure_1_facility_type,
+        latest_blood_pressure_1.district as latest_blood_pressure_1_district,
+        latest_blood_pressure_1.state as latest_blood_pressure_1_state,
+        latest_blood_pressure_1.follow_up_facility_name as latest_blood_pressure_1_follow_up_facility_name,
+        latest_blood_pressure_1.follow_up_date as latest_blood_pressure_1_follow_up_date,
+        latest_blood_pressure_1.follow_up_days as latest_blood_pressure_1_follow_up_days,
+        latest_blood_pressure_1.all_prescription_drugs != latest_blood_pressure_2.all_prescription_drugs as latest_blood_pressure_1_medication_updated,
```

Comment:
Should we call it `latest_blood_pressure_1_medication_titrated`

Reply:
Would like to use different naming here since `titrated` has a different [definition](https://github.com/simpledotorg/simple-server/blob/master/db/views/reporting_patient_states_v06.sql#L181) in our pipeline.